test/testdata/desugar/ensure_without_rescue.rb.cfg.exp

```diff
@@ -18,7 +18,7 @@ subgraph "cluster_::Object#main" {
 
     "bb::Object#main_1" -> "bb::Object#main_1" [style="bold"];
     "bb::Object#main_3" [
-        label = "block[id=3, rubyBlockId=2](<returnMethodTemp>$2: T.untyped, <self>: Object)\l<gotoDeadTemp>$6: TrueClass(true) = true\l<unconditional>\l"
+        label = "block[id=3, rubyBlockId=2](<self>: Object, <returnMethodTemp>$2: T.untyped)\l<gotoDeadTemp>$6: TrueClass(true) = true\l<unconditional>\l"
```

Comment:
Would it be hard to keep the order as it was before?

Reply:
The order before was determined by UnorderedSet iteration order:

https://github.com/sorbet/sorbet/blob/master/cfg/builder/builder_finalize.cc#L411

I don't know how to emulate that. I don't even know if that's supposed to be deterministic; we might have been getting lucky.

Reply:
I can certainly chop this PR up, btw, and land it in stages. I could keep maps there and replace them in a subsequent PR.

Reply:
Oh d'oh, they _are_ sorted. I can re-introduce the sorting and land this later.